forge-gui/res/cardsfolder/upcoming/dross_skullbomb.txt

```diff
@@ -0,0 +1,9 @@
+Name:Dross Skullbomb
+ManaCost:1
+Types:Artifact
+A:AB$ Draw | Cost$ 1 Sac<1/CARDNAME> | SpellDescription$ Draw a card.
+A:AB$ ChangeZone | Cost$ 1 B Sac<1/CARDNAME> | Origin$ Graveyard | Destination$ Hand | TgtPrompt$ Select target creature card in your graveyard | ValidTgts$ Creature.YouCtrl | SorcerySpeed$ True | SubAbility$ DBDraw | SpellDescription$ Return target creature card from your graveyard to your hand. Draw a card. Activate only as a sorcery.
+SVar:DBDraw:DB$ Draw
+AI:RemoveDeck:Random
```

Comment:
Imo DeckNeeds for Black instead would probably be enough as this seems like a rather universal card
(+ creature type if you want to be fully strict but honestly not sure we'll have an AI that would build decks without in the foreseeable future)